docs/providers.md

```diff
@@ -115,15 +125,77 @@ declare(strict_types=1);
 namespace App\BoardGameBlog\Infrastructure\Sylius\Resource;
 
 use App\BoardGameBlog\Infrastructure\Sylius\State\Provider\BoardGameItemProvider;
+use Sylius\Resource\Metadata\AsResource;
+use Sylius\Resource\Metadata\Show;
+use Sylius\Resource\Model\ResourceInterface;
 
-#[Resource]
+#[AsResource]
 #[Show(provider: BoardGameItemProvider::class)]
 final class BoardGameResource implements ResourceInterface
 {
     // [...]
 }
 ```
 
+## Disable providing data
+
+In some cases, you may want not to read data.
+
+For example, in a delete operation, you can implement your custom delete processor without reading it before.
+
+```php
+// src/BoardGameBlog/Infrastructure/Sylius/Resource/BoardGameResource.php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use App\BoardgameBlog\Infrastructure\Sylius\State\Provider\DeleteBoardGameProcessor;
+use Sylius\Resource\Metadata\AsResource;
+use Sylius\Resource\Metadata\Delete;
+use Sylius\Resource\Model\ResourceInterface;
+
+#[AsResource]
+#[Delete(
+    processor: DeleteBoardGameProcessor::class,
+    read: false,
+ )]
+final class BoardGameResource implements ResourceInterface
+{
+    // [...]
+}
+```
+
+```php
+// src/BoardGameBlog/Infrastructure/Sylius/State/Processor/DeleteBoardGameProcessor.php
+
+namespace App\BoardGameBlog\Infrastructure\Sylius\State\Processor;
+
+use Sylius\Resource\Context\Option\RequestOption;
+use Webmozart\Assert\Assert;
+
+final class DeleteBoardGameProcessor implements ProcessorInterface
```

Comment:
Missing `use` for ProcessorInterface, CommandBusInterface, Operation, Context, BoardGameResource, DeleteBoardGameCommand, BoardGameId

Reply:
done